google/resource_compute_snapshot.go

```diff
@@ -112,10 +125,25 @@ func resourceComputeSnapshotCreate(d *schema.ResourceData, meta interface{}) err
 	// It probably maybe worked, so store the ID now
 	d.SetId(snapshot.Name)
 
-	err = computeOperationWait(config, op, project, "Creating Snapshot")
+	err = computeOperationWait(config.clientCompute, op, project, "Creating Snapshot")
 	if err != nil {
 		return err
 	}
+
+	// Now if labels are set, go ahead and apply them
+	if labels := expandLabels(d); len(labels) > 0 {
+		// First, read the remote resource in order to find the fingerprint
+		apiSnapshot, err := config.clientCompute.Snapshots.Get(project, d.Id()).Do()
+		if err != nil {
+			return fmt.Errorf("Eror when reading snapshot for label update: %s", err)
+		}
+
+		// HERE
```

Comment:
Not sure why there is this `HERE` comment

Reply:
oops!